bindgen/src/program.ts

```diff
@@ -67,11 +68,13 @@ const specOption = program
   .argParser((arg, previous: ReadonlyArray<string> = []) => [...previous, parseExistingFilePath(arg)])
   .makeOptionMandatory();
 
+const optInOption = program
+  .createOption("-a, --opt-in <opt-in list>", "Path of the 'opt-in list' specification")
```

Comment:
```suggestion
  .createOption("--opt-in <opt-in list>", "Path of the 'opt-in list' specification")
```

I don't think it is worth adding a short form, and if it was, `-a` doesn't make sense (unless you knew that we used to call this an allowlist)